This fixes #2153 by removing the incorrect permissions. (#2257)
iterative/dvc.org: content/docs/command-reference/config.md

  need to specify private config option values that you don't want to track and
  share with Git (credentials, private locations, etc).

- `--global` - modify a global config file (e.g. `~/.config/dvc/config`) instead
  of the project's `.dvc/config`. Useful to apply config options to all your
  projects.

> - `--system` - modify a system config file (e.g. `/etc/dvc/config`) instead of

Reviewer: Unrelated (and unnecessary) changes. Please roll back.

  > set.

- `cache.shared` - permissions for newly created or downloaded cache files and
  directories. The only accepted value right now is `group`, which makes DVC use
  `664` (rw-rw-r--) for files and `775` (rwxrwxr-x) for directories. This is
  useful when using a
  [shared development server](/doc/use-cases/shared-development-server). The

>   default permissions for cache files is system dependent. In Linux and macOS
>   default permissions of cache is determined using
>   [`os.umask`](https://docs.python.org/3/library/os.html#os.umask). In Windows
>   due to the limitations of
>   [`os.chmod`](https://docs.python.org/3/library/os.html#os.chmod) only
>   read-only bit can be set.

Reviewer: ```suggestion
  default permissions for cache files is system dependent. In Linux and MacOS
  for example, they're determined using
  [`os.umask`](https://docs.python.org/3/library/os.html#os.umask).
```

Reviewer: I removed the Windows part because I think that explanation wasn't accurate. It's OK, we can just skip it.
